pom.xml

```diff
@@ -22,8 +22,8 @@
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>3.15.1</quarkus.version>
-        <org.jooq.version>3.19.22</org.jooq.version>
+        <quarkus.version>3.21.1</quarkus.version>
```

Comment:
Except if you have a strong reason to do it (e.g. you need to upgrade to make it work with 3.21), I would keep 3.15 as, this way, the extension keeps being compatible with 3.15.

Reply:
My app uses Quarkus 3.21.1 and I had the impression that it would not build. But then I think it was because of the versions inconsistencies between q-j and q-j-pro. I will adjust that and try.

Reply:
:heavy_check_mark: